three.js/src/threex/threex-artoolkitcontext-nft.js

```diff
@@ -131,13 +136,15 @@ ARjs.Context.prototype.update = function (srcElement) {
 
     // mark all markers to invisible before processing this frame
     this._arMarkersControls.forEach(function (markerControls) {
-        markerControls.object3d.visible = false
+        if (!markerControls.context.arController.showObject) {
```

Comment:
i think that part missed code was causing to not show the model. An important teminder!